packages/Chem/src/tests/screening-tools.ts

```diff
@@ -60,8 +60,8 @@ category('screening tools', () => {
     await grok.data.detectSemanticTypes(df);
     elementalAnalysis(df, df.getCol('canonical_smiles'), false, false);
     expect(df.columns.length, 29);
-    expect(Array.from(df.row(2).cells).map((c) => c.value).join(''),
-      '1480016COc1ccc2c|c(ccc2c1)C(C)C(=O)OCCCc3cccnc300040203710400.272729992866516126340');
+    expect(Array.from(df.row(40).cells).map((c) => c.value).join(''),
+      '1480010COc1ccc2cc(ccc2c1)C(C)C(=O)OC|CCc3cccnc300040203710400.272729992866516126340');
```

Comment:
Why are we doing it in such a strange manner, by concatenating strings? It's unreadable. It's better to have three expects that are interpretable.